ui/src/app/kvm/components/KVMResourcesCard/KVMResourcesCard.tsx

```diff
@@ -12,32 +13,54 @@ import { MachineListTable } from "app/machines/views/MachineList/MachineListTabl
 type ChartValues = {
   allocated: number;
   free: number;
-  total: number;
-  unit?: string;
 };
 
-type Props = {
+export type Props = {
   className?: string;
   cores: ChartValues;
-  nics: string[];
+  interfaces?: { name: string; virtualFunctions?: ChartValues }[];
   ram: {
+    // All values in B
     general: ChartValues;
-    hugepage?: ChartValues & { pagesize: number };
+    hugepages?: (ChartValues & { pageSize: number })[];
   };
   title?: string;
-  vfs?: ChartValues;
   vms: string[];
 };
 
+/**
+ * Returns a string with the formatted byte value and unit, e.g 1024 => "1KiB"
+ *
+ * @param {number} memory - the memory in bytes
```

Comment:
No need to add the types now, we can follow the [tsdoc](https://github.com/microsoft/tsdoc) standard which excludes types (as the code already documents these).

these can just read:

```
@param memory - the memory in bytes
```
etc.

Reply:
Got it, thanks!